### ISSUE TITLE
Autocomplete field not disabling suggestions

### ISSUE THREAD

**Issue by cmcfadden** [UMN-LATIS/bluesheet#21]
![Screenshot 2023-07-14 at 11 04 37 AM](https://github.com/UMN-LATIS/bluesheet/assets/211655/68933c23-aa3d-449c-acc2-b1636c777527)

(shouldn't show the autofill suggestion)

**Comment by jxjj** [comment]
Might need some help with this one. I'm still seeing it in safari:

![ScreenShot 2023-07-18 at 20 41 37@2x](https://github.com/UMN-LATIS/bluesheet/assets/980170/a37dfcc6-fe73-4c6b-a458-a339e85ecd30)

**Comment by cmcfadden** [comment]
Seems like safari might do this even when autocomplete is off if the field is named in certain ways? To test sometime, we should try renaming the field to some random string.

**Comment by jxjj** [comment]
In #67, I tried a few things:
- giving the input field `name=safari-autofill-fix`
- setting `autocorrect=off`
- setting `autocomplete=new-password` (this didn't seem to make a difference, so I reverted)
- changed the field id from `nameLookup` to just `lookup`

Seems better... but maybe I just cancelled it so many times that it's not occurring for me. I still had to disable autofill contacts in the preferences tho to get it to stop suggesting those. (Maybe it's picking up cues from class names?)

Whatever the case, I don't really see this as a bluesheet issue. So, I'm gonna passive aggressively close this with a labels of `wont-fix` and `switch-browsers`. Feel free to take a crack at it tho! :)
